src/components/organisms/PaneManager/PaneManagerLeftMenu.tsx

```diff
@@ -44,16 +46,16 @@ const PaneManagerLeftMenu: React.FC = () => {
   const rootFileEntry = useMemo(() => fileMap[ROOT_FILE_ENTRY], [fileMap]);
 
   const setLeftActiveMenu = (selectedMenu: LeftMenuSelectionType) => {
+    if (leftDrawerVisible) {
+      dispatch(toggleValidationDrawer(false));
+      dispatch(setLeftMenuSelection(selectedMenu));
+      if (!leftActive) dispatch(toggleLeftMenu());
+      return;
+    }
+
     if (leftMenuSelection === selectedMenu) {
-      if (isStartProjectPaneVisible) {
```

Comment:
@erdkse I remove this since your rework made the left bar no longer visible in the start pane. Does this make sense to you or should I reverse it?